### PR TITLE
Refactor manager orm store

### PR DESCRIPTION
**Pull request by yangjun289519474**
## split orm_stroe.go to several files by diffrents  entries 

#247 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: manager/config/config.go [hunk 1]
```diff
@@ -75,6 +77,26 @@ func New() *Config {
 	}
 }
 
+func (cfg *StoreConfig) CheckValid() error {
```
Comment (gaius-qi) [inline manager/config/config.go:80]: Use func name `Validate`.

File: manager/store/orm/orm_store.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	store.CDNInstance:       NewCDNInstanceStore,
+	store.SecurityDomain:    NewSecurityDomainStore,
+}
+
+func newOrmStore(cfg *config.StoreConfig) (*ormStore, error) {
+	if err := cfg.CheckValid(); err != nil {
+		return nil, err
+	}
+
+	dsn := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?charset=utf8&parseTime=True&loc=Local",
```
Comment (gaius-qi) [inline manager/store/orm/orm_store.go:36]: Assemble with [`url`](https://golang.org/pkg/net/url/) library.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: manager/store/orm/orm_store.go (hunk 2 above)
Comment (yangjun289519474) [reply manager/store/orm/orm_store.go]: fix

File: manager/config/config.go (hunk 1 above)
Comment (yangjun289519474) [reply manager/config/config.go]: fix

**Review by gaius-qi — APPROVED** [review]

**Review by 244372610 — APPROVED** [review]

File: manager/config/config.go [hunk 3]
```diff
@@ -75,6 +77,26 @@ func New() *Config {
 	}
 }
 
-func (cfg *Config) CheckValid() error {
+func (cfg *StoreConfig) Valid() error {
+	if (cfg.Mysql == nil && cfg.Oss == nil) || (cfg.Mysql != nil && cfg.Oss != nil) {
+		return dferrors.Newf(dfcodes.ManagerConfigError, "store config error")
```
Comment (jim3ma) [inline manager/config/config.go:82]: return more accurate error mesage instead of "store config error"

File: manager/config/config.go [hunk 4]
```diff
@@ -75,6 +77,26 @@ func New() *Config {
...
+	}
+
+	if cfg.Mysql != nil {
+		if len(cfg.Mysql.User) == 0 || len(cfg.Mysql.Password) == 0 || cfg.Mysql.Port == 0 || len(cfg.Mysql.IP) == 0 || len(cfg.Mysql.Db) == 0 {
+			return dferrors.Newf(dfcodes.ManagerConfigError, "store config error")
```
Comment (jim3ma) [inline manager/config/config.go:87]: return more accurate error mesage instead of "store config error"

File: manager/config/config.go [hunk 5]
```diff
@@ -75,6 +77,26 @@ func New() *Config {
...
+		}
+
+		return nil
+	}
+
+	if cfg.Oss != nil {
+		return dferrors.Newf(dfcodes.ManagerConfigError, "store config error")
```
Comment (jim3ma) [inline manager/config/config.go:94]: return more accurate error mesage instead of "store config error"

**Review by jim3ma — DISMISSED** [review]
LGTM

**Review by 244372610 — APPROVED** [review]

**Review by gaius-qi — APPROVED** [review]

**Review by 244372610 — APPROVED** [review]

**Pull request merged**